bundle/bundle.go

```diff
@@ -62,6 +62,49 @@ func (m *Manifest) Init() {
 	}
 }
 
+// Equal returns true if is semantically equivalent to other.
+func (m Manifest) Equal(other Manifest) bool {
+
+	// This is safe since both are passed by value.
+	m.Init()
+	other.Init()
```

Comment:
Yeah, I think goal was to make the zero value valid--not sure whether it's useful but that's how we've tried to treat it elsewhere AFAIK. Trying to avoid changing that here.